src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseGradingService.java

```diff
@@ -728,57 +742,44 @@ private boolean createFeedbackForDuplicateTests(Result result, ProgrammingExerci
      * Update the score given the positive tests score divided by all tests' score.
      * Takes weight, bonus multiplier and absolute bonus points into account.
      * All tests in this case do not include ones with visibility=never.
-     * @param result                     of the build run.
-     * @param allTestCases               of a given programming exercise.
-     * @param successfulTestCases        test cases with positive feedback.
-     * @param staticCodeAnalysisFeedback of a given programming exercise.
-     * @param programmingExercise        the given programming exercise.
-     * @param hasDuplicateTestCases      indicates duplicate test cases.
+     *
```

Comment:
Missing JavaDoc for `scoreCalculationData`, same for the other methods below.

Reply:
Updated comments in https://github.com/ls1intum/Artemis/pull/5994/commits/901f38ceacb9865db6ab72b61425249de3f80a12

Reply:
Why did you completely remove the `@param` parts instead of updating them to match the new actual parameters?

Reply:
I didn't really see the points of those comments

Reply:
The code style checks say that they should be kept: https://github.com/ls1intum/Artemis/actions/runs/3948049084/jobs/6757586390

Reply:
Resolved the checks by readding some comments https://github.com/ls1intum/Artemis/pull/5994/commits/930fafa5bed5cc9c7f5d77a7b9f8f1fba5251af9